lib/tasks/populate_number_of_samples_per_pool.rake

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+namespace :number_of_samples_per_pool do
+  desc 'Populate number of samples per pool column in request_metadata table for a given submission ID'
+
+  task :populate, %i[samples_per_pool submission_id] => :environment do |_, args|
+    # TODO: Replace it with the pre-determined number of samples per pool
+    args.with_defaults(samples_per_pool: 96)
```

Comment:
I think 96 is not a good default value for samples_per_pool. I would make sure this value is specified or I would make it fail instead of using a default.